roles/openshift_repos/vars/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 known_openshift_deployment_types: ['origin', 'online', 'enterprise']
+known_openshift_product_types: ['origin', 'openshift', 'atomic-enterprise']
```

Comment:
I think this should be limited to `openshift` and `atomic-enterprise` as those are the valid service name prefixes.